src/main/java/org/team199/lib/MotorControllerFactory.java

```diff
@@ -80,4 +83,9 @@ public static CANSparkMax createSparkMax(int id) {
 
     return spark;
   }
+  public static UsbCamera configureCamera(int port) {
```

Comment:
This should be moved to a different class. Perhaps a new CameraFactory class or a CameraSubsystemBase class.